speechbrain/dataio/dataio.py

```diff
@@ -28,6 +31,50 @@
 logger = logging.getLogger(__name__)
 
 
+def load_audio_pyav(uri, num_frames=-1, frame_offset=0, normalize=True):
+    """Mimics torchaudio.load using PyAV which bundles ffmpeg.
+
+    Arguments
+    ---------
+    uri: str
+        Path to file
+    num_frames: int
+        Number of frames to load. Default (-1) loads all frames
+    frame_offset: int
+        Frame from which to start loading. Default (0) loads all frames
+    normalize: bool
+        Whether to normalize output to float32 if int32
+    """
+
+    raw_buffer = io.BytesIO()
+    dtype = None
+    sample_rate = None
+    channels = None
+
+    with av.open(uri, mode="r", metadata_errors="ignore") as container:
+        for frame in container.decode(audio=0):
+            sample_rate = frame.rate
+            layout = frame.layout
+            channels = len(layout.channels)
+            array = frame.to_ndarray()
+            dtype = array.dtype
+
+            raw_buffer.write(array)
+
+    audio = np.frombuffer(raw_buffer.getbuffer(), dtype=dtype)
+    audio = np.reshape(audio, [channels, -1], order="F")
+
+    if dtype.kind == "i" and normalize:
+        audio = audio.astype(np.float32) / np.iinfo(dtype).max
+
+    if num_frames >= 0:
+        audio = audio[:, frame_offset : frame_offset + num_frames]
+    elif frame_offset > 0:
+        audio = audio[:, frame_offset:]
+
+    return torch.tensor(audio), sample_rate
```

Comment:
I think using `torch.from_numpy` method instead of `torch.tensor` would be better (in terms of readability/torch.from_numpy infers the dtype of your audio file while torch.tensor is an alias of torch.FloatTensor)